types/k6/net/grpc.d.ts

```diff
@@ -7,7 +7,7 @@
 export interface Response {
     status: number;
 
-    message: string;
+    message: object;
```

Comment:
Please, add a test for this change.

Reply:
Sorry, no idea how do I test for this. Does [this line](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57368/files#diff-c5fc3e839decd92e84fec26d34351fdae689002a05f09e0474a999e3a8b43f6dR25) cover the case?

Reply:
Should work replacing the line with

```
response.message; // $ExpectType object
```

Reply:
Thank you for your guidance.
 Fixed
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57368/commits/69023ae288075b64abae852410b437965dd85273